scripts/serve/defaultWebpackConfig.js

```diff
@@ -0,0 +1,25 @@
+const path = require('path');
+const fs = require('fs');
+
+const isFile = filePath => (fs.existsSync(filePath) && !fs.lstatSync(filePath).isDirectory());
+
+const resolve = (filePath) => {
+  if (isFile(filePath)) {
+    // eslint-disable-next-line global-require, import/no-dynamic-require
+    return require(filePath);
+  }
+  return undefined;
+};
+
+const defaultWebpackConfig = () => {
+  // First try to find the local to process.cwd webpack config
+  const localConfig = resolve(path.resolve(process.cwd(), 'webpack.config.js'));
+  if (localConfig) {
+    return localConfig;
+  }
+
+  // If that is not found look for the terra-dev-site webpack config.
+  return resolve(path.resolve(process.cwd(), 'node_modules', 'terra-dev-site', 'src', 'webpack', 'webpack.config.js'));
+};
```

Comment:
I think this should have a third fallback,
 1- check local
 2- check for terra-dev-site
 3- use toolkit's

Reply:
Falling back to Toolkits would be inappropriate because it requires an entry to be added.

Reply:
hmmmmmm I missed that part.